products/debian12/product.yml

```diff
@@ -35,3 +35,5 @@ platform_package_overrides:
   pam: libpam-runtime
   shadow: login
   sssd: sssd-common
+  chrony_conf_path: "/etc/chrony/chrony.conf"
+  chrony_d_path: "/etc/chrony/chrony.d"
```

Comment:
the CI fails because you will have to update the product stability data

Reply:
I have updated the product stability data (and rebased onto current master).